src/CMakeLists.txt

```diff
@@ -640,7 +640,7 @@ find_package(Doxygen)
 if(DOXYGEN_FOUND)
     configure_file(${CMAKE_CURRENT_SOURCE_DIR}/Doxyfile.in ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile @ONLY)
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../doc )
-    add_custom_target(doc
+    add_custom_target(pangolin_doc
```

Comment:
You should also update the docstring above that currently reads `(make doc)`

Reply:
done